### PR TITLE
MCReader: Fixed Lasted manga

### LINKED ISSUE #12261: Mcreader.net: HTTP error 404
### Source information

Mcreader.net 1.3.12

### Steps to reproduce

It's not working it's opening in the web view but not working in normal mode

### Expected behavior

When opening view normally

### Actual behavior

It's shows https error 404

### Tachiyomi version

0.13.4

### Android version

Android 12

### Other details

Screenshot_20220620_213758_afd66b7916d19a60815f56273408b73e

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by pizidavi**
Closes #12261

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt [hunk 1]
```diff
@@ -20,7 +20,7 @@ import java.util.concurrent.TimeUnit
 class MangaRawClub : ParsedHttpSource() {
 
     override val id = 734865402529567092
-    override val name = "mcreader.net"
+    override val name = "MCReader"
```
Comment (arkon) [inline src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt:23]: Is it really worth changing the name again?

Comment (pizidavi) [reply src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt:23]: Technically no, but the correct name is "MCReader" not its url, if it is not complicated I would like it changed

Comment (arkon) [reply src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt:23]: It'll result in orphaned downloads if the folder isn't manually renamed.

Comment (pizidavi) [reply src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt:23]: Ok, I'll revert

**Pull request merged**